### PR TITLE
use libgcc,libstdcxx pins

### PR DESCRIPTION
**Pull request by jayfurmanek**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Anaconda just updated toolchains and associated libraries. One of the changes is breaking out libgomp into its own library.
This PR addresses XGBoost by using the pins created in: https://github.com/open-ce/open-ce/pull/314

https://github.com/open-ce/open-ce/issues/315

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/master/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -18,13 +18,17 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - libgcc-ng {{ libgcc }}
```
Comment (npanpaliya) [inline recipe/meta.yaml:21]: These pins are added only for x86 in conda_build_config.yaml. So, we need #[x86_64] here too.

**Review by npanpaliya — COMMENTED** [review]
PR to add pins is raised against r1.1 branch, but this PR is against master.

File: recipe/meta.yaml (hunk 1 above)
Comment (jayfurmanek) [reply recipe/meta.yaml:21]: I updated https://github.com/open-ce/open-ce/pull/314 to include the pin for ppc64le as well.

Comment (jayfurmanek) [reply recipe/meta.yaml:21]: Also we don't have a branch for 1.1.x, only tags. So we can push this to master, then branch it when needed. We may want to control these pins on master going forward as well anyway.

**Comment by jayfurmanek** [comment]
On hold. Anaconda pulled the new libgcc and toolchain packages for now.

**Review by bnemanich — APPROVED** [review]

**Pull request merged**